uber/configspec.ini

```diff
@@ -176,15 +176,9 @@ hours_for_refund = integer(default=24)
 # This setting determines how many days in advance we begin tracking setup shifts.
 setup_shift_days = integer(default=5)
 
-# There are two separate deadlines for custom badges; the one after which it's
-# too late for attendees to edit their custom badge submissions, and the one
-# where it's too late for an admin to shift badge numbers on the backend.  The
-# former is controlled by the PRINTED_BADGE_DEADLINE field, which is what we
-# advertise in our emails and such.
-#
-# Once the shift option is set to False, custom badge numbers are never changed
-# automatically.  This is a bool instead of a date since we don't usually
-# know the cutoff in advance.
+# Some events wish to be able to freely edit badge numbers throughout the year,
```

Comment:
I noticed this while looking around this file -- the documentation still talked about the 'old way' of doing things where we would set shift_custom_badges to false sometime during the year. This was an issue so many years in a row that we finally just set the badge shifting to not work after the printed badge deadline.